.github/workflows/scancode-release.yml

```diff
@@ -447,22 +457,94 @@ jobs:
           name: source_app
           path: dist
 
-      - name: Download a single artifact macos_app
+      - name: Download a single artifact macos_app for python 3.7
```

Comment:
This is too ugly and repeats a lot, maybe there is some way we can write this in a loop? Couldn't find anything useful AFAIK in https://github.com/actions/upload-artifact#readme

Reply:
From @pombredanne: this might be ugly but it is probably more maintainable so this is ok